From 888e5149bdb810e67996828bb26955a57a482d4c Mon Sep 17 00:00:00 2001
From: Maxime Ripard <maxime@cerno.tech>
Date: Tue, 14 Jan 2020 13:37:27 +0100
Subject: [PATCH] drm/vc4: crtc: Use local chan variable

The vc4_crtc_handle_page_flip already has a local variable holding the
value of vc4_crtc->channel, so let's use it instead.

Signed-off-by: Maxime Ripard <maxime@cerno.tech>
---
 drivers/gpu/drm/vc4/vc4_crtc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/gpu/drm/vc4/vc4_crtc.c
+++ b/drivers/gpu/drm/vc4/vc4_crtc.c
@@ -816,7 +816,7 @@ static void vc4_crtc_handle_page_flip(st
 		 * underruns. This can be seen when reconfiguring the CRTC.
 		 */
 		if (vc4->hvs)
-			vc4_hvs_unmask_underrun(dev, vc4_crtc->channel);
+			vc4_hvs_unmask_underrun(dev, chan);
 	}
 	spin_unlock_irqrestore(&dev->event_lock, flags);
 }
